### ISSUE TITLE
Cannot import name 'FieldDoesNotExist' from 'django.db.models.fields'

### ISSUE THREAD

**Issue by AbdealiJK** [carltongibson/django-filter#1230]
When using the latest development django, I get the error:
```
File "venv/lib/python3.7/site-packages/django_filters/utils.py", line 10, in <module>
    from django.db.models.fields import FieldDoesNotExist
ImportError: cannot import name 'FieldDoesNotExist' from 'django.db.models.fields' (venv/lib/python3.7/site-packages/django/db/models/fields/__init__.py)
```

Caused by:
https://docs.djangoproject.com/en/dev/releases/3.1/
> The compatibility import of django.core.exceptions.FieldDoesNotExist in django.db.models.fields is removed.

While that django version is in dev - I thought I'd raise it here

**Comment by carltongibson** [comment]
Super.  Thanks for the report.

**Comment by carltongibson** [comment]
Fixed in a5b4d0299ec04dcb103dc37291001fb268107c66. Will be part of v2.3 today.
